### PR TITLE
Custom model created for User

### PR DESCRIPTION
**Pull request by theyashshahs**

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+from .managers import MyUserManager
+
+
+class MyUser(AbstractBaseUser, PermissionsMixin):
+    first_name = models.CharField(_('first name'), max_length=30, blank=True)
```
Comment (rossoskull) [inline accounts/models.py]: We don't need to specify `first_name`, `last_name`, `is_active`, `email`, `is_staff` fields here, AbstractBaseUser provides us with them

File: accounts/models.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+from .managers import MyUserManager
+
+
+class MyUser(AbstractBaseUser, PermissionsMixin):
```
Comment (rossoskull) [inline accounts/models.py]: Can we also rename this model to like `Account`

Comment (theyashshahs) [reply accounts/models.py]: okay, I will work on this

**Review by rossoskull — APPROVED** [review]

File: accounts/models.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+from .managers import AccountManager
```
Comment (Monal5031) [inline accounts/models.py:6]: :angry: relative imports @rossoskull @yashshah2820

**Pull request merged**